bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/MainScreen.kt

```diff
@@ -0,0 +1,46 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import net.gini.android.bank.sdk.exampleapp.R
+
+class MainScreen {
+
+    fun assertWelcomeTitle(): MainScreen {
+        onView(withId(R.id.tv_welcomeToGini)).check(matches(isDisplayed()))
```

Comment:
[Suggestion] It seems to me that this test checks if a view with id `R.id.tv_welcomeToGini` is displayed in the view 🤔. If that's true, maybe the title of the test should be something like `checkWelcomeTitleIsDisplayed` ? 

Otherwise it seems like the test checks what the welcome title is...

Reply:
Agreed, I will update the title of test 😄